settings/LoanPolicyForm.js

```diff
@@ -122,10 +122,14 @@ class LoanPolicyForm extends React.Component {
 
     const openingTimeOffset = (allValues.loansPolicy && allValues.loansPolicy.openingTimeOffset);
     const openingTimeOffsetIsVisible = openingTimeOffset && openingTimeOffset.intervalId && !openingTimeOffset.duration;
+    const openingTimeOffsetIsDefault = openingTimeOffset && openingTimeOffset.duration && !openingTimeOffset.intervalId;
```

Comment:
Explain why do we need it and to if statements with `this.props.change`?

Reply:
The default units for this field are 'Hours', that is why when we select a value it automatically measures it in  'Hours'

Reply:
Sorry, but do you have any concerns still?